grammars/javascript.cson

```diff
@@ -431,6 +431,10 @@
     'name': 'support.constant.dom.js'
   }
   {
+    'match': '(?<=(?:[a-zA-Z_$0-9]\\.))([a-zA-Z_$][a-zA-Z_$0-9]*)'
```

Comment:
This seems a bit out-of-place since the surrounding capture rules all start with `support`.  Is there by any chance a better place to put it?